scripts/kb-scripts/support_scripts/scs_writer.py

```diff
@@ -6,8 +6,8 @@
     "node/-/not_define": ["sc_node"],
 
     "node/const/general_node": ["sc_node"],
-    "node/const/terminal": ["sc_node_abstract"],
-    "node/const/struct": ["sc_node_struct"],
+    "node/const/terminal": ["sc_node_superclass"],
```

Comment:
terminal element does not have any elements belonging to it, why'd you replace it with superclass?

Reply:
KBE has this type